wordpress-mobile/MediaPicker-iOS
# Make sure the refresh group flag is updated correctly on all states.

Closes #124 

How to test:
 - Open the MediaPicker app
 - Select a group with zero photos
 - See if the camera capture option still shows up
